torch/hub.py

```diff
@@ -178,6 +182,8 @@ def _get_cache_or_reload(github, force_reload, verbose=True, skip_validation=Fal
     # and inspect name from it.
     # To check if cached repo exists, we need to normalize folder names.
     repo_dir = os.path.join(hub_dir, '_'.join([repo_owner, repo_name, normalized_br]))
+    # Check that the repo is in the trusted list
+    _check_repo('_'.join([repo_owner, repo_name, normalized_br]), trust_repo=trust_repo, calling_fn=calling_fn)
```

Comment:
Thoughts on *not* including the branch name?

I feel like it's fair to assume that user would trust an entire repo, and not just some branches of it, but happy to hear your thoughts.

If we're keeping `'_'.join([repo_owner, repo_name, normalized_br]` them let's just declare it above because it's also used to defined `repo_dir`.

Reply:
yes I agree, `{repo_owner}_{repo_name}` is sufficient